module_evaluations/TCR-32_2023-09-06.MD

```diff
@@ -0,0 +1,79 @@
+# Module acceptance criteria template
+
+## How to use this form
+When performing a technical evaluation of a module, create a copy of this document and use the conventions below to indicate the status of each criterion.  The evaluation results should be placed in the [module_evaluations](https://github.com/folio-org/tech-council/tree/master/module_evaluations) directory and should conform to the following naming convention: `{JIRA Key}_YYYY-MM-DD.MD`, e.g. `TCR-1_2021-11-17.MD`.  The date here is used to differentiate between initial and potential re-evaluation(s).  It should be the date when the evaluation results file was created.
+
+* [x] ACCEPTABLE
+* [x] ~INAPPLICABLE~
+* [ ] UNACCEPTABLE
+  * comments on what was evaluated/not evaluated, why a criterion failed
+
+## [Criteria](https://github.com/folio-org/tech-council/blob/7b10294a5c1c10c7e1a7c5b9f99f04bf07630f06/MODULE_ACCEPTANCE_CRITERIA.MD)
+
+## Shared/Common
+* [x] Uses Apache 2.0 license
+* [x] Module build MUST produce a valid module descriptor
+* [ ] Module descriptor MUST include interface requirements for all consumed APIs
+  * missing `edge-fqm`
```

Comment:
That's strange, a UI module should not depend upon an edge module, especially as they generally don't provide interfaces (because they are kinda outside FOLIO)

Reply:
Nope, I just misunderstood https://dev.folio.org/reference/api/endpoints/.